packages/terra-badge/lib/Badge.scss

```diff
@@ -0,0 +1,81 @@
+@import 'node_modules/terra-mixins/src/terra-mixins';
+@import './variables';
+@import './mixins';
+
+.terra-Badge {
+  border-radius: $terra-badge-border-radius;
+  display: $terra-badge-display;
+  font-size: map-get($terra-badge-font-sizes, small);
+  font-weight: $terra-badge-font-weight;
+  line-height: $terra-badge-line-height;
+  min-width: 1.65em;
+  padding-bottom: $terra-badge-padding-bottom;
+  padding-left: $terra-badge-padding-right;
+  padding-right: $terra-badge-padding-left;
```

Comment:
Looks like this needs to be switched over to use the bidi mixin:
`@include terra-padding-end($terra-badge-padding-right);`